### PR TITLE
Add the feature doc for `proposal-removal`

### PR DESCRIPTION
**Pull request by shannynalayna**
This PR updates the man pages based on the `just copy-docs` command. Also adds the feature doc for `proposal-removal`.

### CODE REVIEW

File: community/planning/proposal_removal.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+## Prior art
+[prior-art]: #prior-art
+
+This feature is influenced by the `splinter-circuit-abandon` command, used to
+delete a Splinter circuit. The process to remove a circuit proposal does follow
+a similar pattern to the abandon command, as each is performed by a single node
+and doesn't cause any state changes for the other circuit and circuit proposal
+members. Abandoning a circuit does not completely remove the circuit from the
+node’s state, but requesting to remove a circuit proposal will. Removing a
```
Comment (agunde406) [inline community/planning/proposal_removal.md:164]: You do not need the last sentence.

File: docs/0.5/references/cli/splinter-circuit-remove-proposal.1.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+% SPLINTER-CIRCUIT-REMOVE-PROPOSAL(1) Cargill, Incorporated | Splinter Commands
```
Comment (agunde406) [inline docs/0.5/references/cli/splinter-circuit-remove-proposal.1.md:1]: Please add a commit to add a link to this page the CLI Command Reference page

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: community/planning/proposal_removal.md (hunk 1 above)
Comment (shannynalayna) [reply community/planning/proposal_removal.md]: Removed that sentence.

File: docs/0.5/references/cli/splinter-circuit-remove-proposal.1.md (hunk 2 above)
Comment (shannynalayna) [reply docs/0.5/references/cli/splinter-circuit-remove-proposal.1.md:1]: Added a commit to add a link to this as well as the circuit-delete commands

**Review by agunde406 — APPROVED** [review]

**Pull request merged**